projects/sbb-esta/angular-public/file-selector/src/_file-selector.scss

```diff
@@ -11,18 +11,28 @@
 
 @mixin fileSelectorButton() {
```

Comment:
Without investigating further, your changes somehow messed up disabled state in public component (see https://angular.app.sbb.ch/latest/public/components/file-selector).

Reply:
your version with wrong disabled styling: https://angular-staging.app.sbb.ch/352/public/components/file-selector

Reply:
The problem still occurs in public: the disabled state should have a border, see https://angular.app.sbb.ch/latest/public/components/file-selector for comparison. Probably` &:not(.sbb-file-selector-disabled)` is still required. for @include secondaryButton.

Reply:
Interesting because it works locally on my laptop. I'll take a look.

Reply:
it can be a css bleeding issue, which depends on browser and usage. maybe that was the reason for the :not selector before